docs/api.md

```diff
@@ -247,6 +247,32 @@ test('waiting for an Banana to be ready', async () => {
 });
 ```
 
+## `wait`
+
+Defined as:
+
+```jsx
+function wait(
+  expectation: () => Promise<{}> = () => Promise.resolve({}),
+  timeout: number = 4500,
+  interval: number = 50
+): Promise<{}> {}
+```
+
+Waits for non-deterministic periods of time until your expectation function succeed or times out. `wait` is a small wrapper
+around the [`wait-for-expect`](https://github.com/TheBrainFamily/wait-for-expect) module.
+
+```jsx
+import { render, wait } from 'react-testing-library';
+
+test('waiting for an Banana to be ready', async () => {
+  const { getByText, toJSON } = render(<Banana />);
+
+  await wait(expect(getByText('Loading...')..toThrow()));
```

Comment:
I believe this won't work, because `wait` expects returning a promise, and neither `expect` nor `getByText` does that.